packages/nakama-js/socket.ts

```diff
@@ -250,43 +387,54 @@ export interface StatusPresenceEvent {
 }
 
 /** Stop receiving status updates for some set of users. */
-export interface StatusUnfollow {
+interface StatusUnfollow {
   status_unfollow: {user_ids: string[];};
 }
 
 /** Set the user's own status. */
-export interface StatusUpdate {
+interface StatusUpdate {
   status_update: {status?: string;};
 }
 
 /** A socket connection to Nakama server. */
 export interface Socket {
   // Connect to the server.
   connect(session: Session, createStatus: boolean): Promise<Session>;
+
   // Disconnect from the server.
   disconnect(fireDisconnectEvent: boolean): void;
-  // Send message to the server. This method remains in the API for backwards compatibility.
-  // We recommend that you use the other socket-based methods below for improved
-  // type checking and code readability.
-  send(message: ChannelJoin | ChannelLeave | ChannelMessageSend |
-    ChannelMessageUpdate | ChannelMessageRemove | CreateMatch | JoinMatch |
-    LeaveMatch | MatchDataSend | MatchmakerAdd | MatchmakerRemove | Rpc |
-    StatusFollow | StatusUnfollow | StatusUpdate): Promise<any>;
+
+  // Accept a request to join.
+  acceptPartyMember(party_id : string, presence : Presence) : Promise<void>;
 
   /// Join the matchmaker pool and search for opponents on the server.
   addMatchmaker(query : string, minCount : number, maxCount : number,
     stringProperties? : Record<string, string>, numericProperties? : Record<string, number>)
-    : Promise<MatchmakerMatched>;
+    : Promise<MatchmakerTicket>;
+
+  // Begin matchmaking as a party.
+  addMatchmakerParty(party_id: string, query : string, min_count : number, max_count : number,
+    string_properties? : Record<string, string>, numericProperties? : Record<string, number>)
+    : Promise<void>;
```

Comment:
@zyro I think I had returned `Promise<void>` in this client and the dotnet client because I wasn't seeing the ticket returned with the `addMatchmakerParty` cid. Is it possible we missed this in our discussions on cid grouping? I can't think of a reason this should be different from `addMatchmaker`.

Reply:
Confirmed in client logs that the client doesn't get any ticket information in response to the cid associated with this sent message:
```
[INFO] Sending message {"cid":"3","party_matchmaker_add":{"party_id":"219b9bae-a95e-4f83-a8ad-e5b4928411e0.nakama1","max_count":2,"min_count":2,"query":"*"}}
[INFO] Received message {"cid":"3"}
```

Reply:
Andrei put in a server change for this for 3.3.1 and I will update the signatures on the js and dotnet client.

Reply:
Done.